common/src/main/java/org/apache/celeborn/common/haclient/MasterNotLeaderException.java

```diff
@@ -26,18 +26,18 @@ public class MasterNotLeaderException extends IOException {
 
   public static final String LEADER_NOT_PRESENTED = "leader is not present";
 
-  public MasterNotLeaderException(String currentPeer, String suggestedLeaderPeer) {
+  public MasterNotLeaderException(String currentPeer, String suggestedLeaderPeer, Throwable cause) {
     super(
-        "Master:"
-            + currentPeer
-            + " is not the leader. Suggested leader is"
-            + " Master:"
-            + suggestedLeaderPeer
-            + ".");
+        "Master: currentPeer may not the leader. Suggested leader is Master: suggestedLeaderPeer.",
```

Comment:
"Master: currentPeer may not the leader. Suggested leader is Master: " + suggestedLeaderPeer + ".", ?